lib/devise_ldap_authenticatable/ldap/connection.rb

```diff
@@ -20,15 +20,20 @@ def initialize(params = {})
         @attribute = ldap_config["attribute"]
         @allow_unauthenticated_bind = ldap_config["allow_unauthenticated_bind"]
 
-        @ldap_auth_username_builder = params[:ldap_auth_username_builder]
+        @ldap_auth_username_builder = ::Devise.ldap_auth_username_builder
 
+        @user_lookup_attribute = ldap_config["user_lookup_attribute"] || 'mail'
+        @group_lookup_attribute = ldap_config["group_lookup_attribtue"] || 'memberof'
         @group_base = ldap_config["group_base"]
         @check_group_membership = ldap_config.has_key?("check_group_membership") ? ldap_config["check_group_membership"] : ::Devise.ldap_check_group_membership
+        @check_group_membership_without_admin = ldap_config.has_key?("check_group_membership_without_admin") ? ldap_config["check_group_membership_without_admin"] : ::Devise.ldap_check_group_membership_without_adminkj
         @required_groups = ldap_config["required_groups"]
         @required_attributes = ldap_config["require_attribute"]
 
+        @user_lookup_attribute = ldap_config["user_lookup_attribute"] || 'mail'
+        @group_lookup_attribute = ldap_config["group_lookup_attribtue"] || 'memberof'
```

Comment:
@user_lookup_attribute and @group_lookup_attribute are declared above as well. please remove one of them